### ISSUE TITLE
[TODO] Verify that the games' save/load feature is working before release

### ISSUE THREAD

**Issue by shlomif** [shlomif/PySolFC#131]
The conversion to https://www.attrs.org/en/stable/ may have broken it.

**Comment by lufebe16** [comment]
Indeed, restoring of games failed due to refactoring with attrs. Fixed it on occasion, commit 97267e9.

**Comment by lufebe16** [comment]
@shlomif 
This worked so far.    	

But when I think about it deeper, a big black hole opens, that swallows all my statistics and all my games collected over years! - We would have to provide migration code.

The easier way to get back to a to a running solution would be to revert these changes, at least regarding moves,statistics and gstatistics structs.

**Comment by lufebe16** [comment]
New and old formats seem to be compatible on import, only the code throws an assert. So this could be fixed by ignoring some checks. Then old games from the users collections are still readable. Commit 92f2d5f.
